modules/otel-receivers-prometheus-remote-write-receiver.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * observability/otel/otel-collector/otel-collector-receivers.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="otel-receivers-prometheus-remote-write-receiver_{context}"]
+= Prometheus Remote Write Receiver
+
+[role="_abstract"]
+The Prometheus Remote Write Receiver receives metrics from Prometheus via the Remote Write protocol and converts them to the OpenTelemetry format. This receiver supports only the Prometheus Remote Write v2 protocol.
+
+:FeatureName: The Prometheus Remote Write Receiver
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled Prometheus Remote Write Receiver
+[source,yaml]
+----
+# ...
+  config:
+    receivers:
+      prometheusremotewrite:
+        endpoint: 0.0.0.0:9009 # <1>
+    # ...
+    service:
+      pipelines:
+        metrics:
+          receivers: [prometheusremotewrite]
+# ...
+----
+<1> The endpoint where the receiver listens for Prometheus Remote Write requests.
+
+The following are the prerequisites for using the Prometheus Remote Write Receiver with Prometheus:
+
+* Prometheus is started with the metadata WAL records feature flag enabled:
++
+[source,yaml]
+----
+./prometheus --config.file config.yml --enable-feature=metadata-wal-records
```

Comment:
At RH we don't support running the "RAW" prometheus. Hence we should rather mention what needs to be enabled and link to the prometheus docs.